swig/python/gdal-utils/osgeo_utils/samples/classify.py

```diff
@@ -62,13 +67,15 @@ def doit(src_filename, dst_filename):
     gdal_array.SaveArray(dst_image, dst_filename)
 
 
-def main(argv=sys.argv):
-    src_filename = 'utm.tif'
-    dst_filename = 'classes.tif'
+def main(argv=sys.argv, src_filename=None, dst_filename=None):
+    # src_filename = 'utm.tif'
+    # dst_filename = 'classes.tif'
     if len(argv) > 1:
         src_filename = argv[1]
     if len(argv) > 2:
         dst_filename = argv[2]
+    if not dst_filename:
```

Comment:
src_filename should probably be tested too

Reply:
good idea